frontend/server/src/main/java/org/pytorch/serve/wlm/ModelManager.java

```diff
@@ -152,6 +158,27 @@ private ModelArchive createModelArchive(
         return archive;
     }
 
+    private void setupModelDependencies(Model model) throws IOException, InterruptedException {
+        String requirementsFile =
+                model.getModelArchive().getManifest().getModel().getRequirementsFile();
+
+        if (configManager.getInstallPyDepPerModel() && requirementsFile != null) {
+            Path requirementsFilePath =
+                    Paths.get(model.getModelDir().getAbsolutePath(), requirementsFile);
+
+            String packageInstallCommand =
+                    "pip install -U -t "
```

Comment:
Does this work if a custom python interpretor has been specified in the config? Wondering if `<python_executable> -p pip -U -t ...` would be more appropriate

Reply:
Done.